packages/react-core/src/components/Tooltip/examples/TooltipBasic.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Button, Tooltip } from '@patternfly/react-core';
+
+export const TooltipBasic: React.FunctionComponent = () => (
+  <div style={{ margin: '100px' }}>
+    <Tooltip
+      content={
+        <div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis.
+        </div>
+      }
+    >
+      <Button tabIndex={0}>I have a tooltip!</Button>
```

Comment:
Just caught this, but we can remove the `tabIndex` from the Button since it should be focusable by default.